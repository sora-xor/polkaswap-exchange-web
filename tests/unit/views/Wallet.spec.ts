// import { shallowMount, createLocalVue } from '@vue/test-utils'

// import Wallet from '@/views/Wallet.vue'
// import { SoramitsuElementsImport, WalletImport, TranslationMock } from '../../utils'

// const localVue = createLocalVue()
// SoramitsuElementsImport(localVue)
// WalletImport(localVue)

// describe('Wallet.vue', () => {
//   beforeEach(() => {
//     TranslationMock(Wallet)
//   })

//   it('should renders correctly', () => {
//     const wrapper = shallowMount(Wallet, { localVue })
//     expect(wrapper.element).toMatchSnapshot()
//   })
// })

describe('Wallet test', () => {
  test('Temporary test', () => {
    expect(true).toEqual(true);
  });
});
