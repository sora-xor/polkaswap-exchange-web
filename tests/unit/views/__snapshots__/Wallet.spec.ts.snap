// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wallet.vue should renders correctly 1`] = `
<div
  class="wallet-container s-flex"
>
  <sora-neo-wallet-stub />
</div>
`;
